src/components/Input/BaseInput/BaseInput.scss

```diff
@@ -82,7 +81,7 @@ $border-radius: 2px;
 
     // Reset Firefox invalid required input style
     &:invalid {
-        box-shadow: none;
```

Comment:
We want to reset the default style provided by Firefox. What is the style added by Firefox when the input is invalid? A `border` or a `box-shadow`?

Reply:
I did some research and added both `box-shadow: none` and `border: none`